elyra/__init__.py

```diff
@@ -41,5 +42,6 @@ def load_jupyter_server_extension(nb_server_app):
         (url_path_join(web_app.settings['base_url'], r'/api/metadata/%s' % (namespace_regex)), MetadataHandler),
         (url_path_join(web_app.settings['base_url'], r'/api/metadata/%s/%s' % (namespace_regex, resource_regex)),
          MetadataNamespaceHandler),
+        (url_path_join(web_app.settings['base_url'], r'/api/pipeline/export'), PipelineExportHandler),
```

Comment:
This endpoint needs to be added to the swagger docs (JSON and YAML) in https://github.com/elyra-ai/elyra/tree/master/elyra/api.

Reply:
Just seeing that scheduler is not documented at all.  I've created Issue #434 for this.

Reply:
I just remembered we chose not to document some endpoints and have closed #434.  This comment can be ignored.